### PR TITLE
Add rest-source-authorization service and app to the stack

### PR DESCRIPTION
**Pull request by nivemaham**
Adds rest-source-authorization service and app to the stack

### CODE REVIEW

File: dcompose-stack/radar-cp-hadoop-stack/lib/perform-install.sh [hunk 1]
```diff
@@ -113,7 +113,7 @@ ensure_variable 'topics=' "${COMBINED_RAW_TOPIC_LIST}" etc/hdfs-connector/sink-h
 
 echo "==> Configuring Management Portal"
 sudo-linux bin/radar-docker up -d --build radarbase-postgresql
-sleep 10
+sleep 20
```
Comment (blootsvoets) [inline dcompose-stack/radar-cp-hadoop-stack/lib/perform-install.sh]: Perhaps add a poll in the multi-dB-init instead? Like every second check if Postgres is ready, and the run the rest of the script?

File: dcompose-stack/radar-cp-hadoop-stack/docker-compose.yml [hunk 2]
```diff
@@ -712,16 +712,20 @@ services:
...
 
 
-  managementportal-postgresql:
-    image: postgres:${POSTGRES_VERSION}
+  radarbase-postgresql:
+    build:
+      context: ./images/postgres
+      args:
+        POSTGRES_VERSION: ${POSTGRES_VERSION}
+    image: postgres:radarbase-${POSTGRES_VERSION}
```
Comment (blootsvoets) [inline dcompose-stack/radar-cp-hadoop-stack/docker-compose.yml]: Perhaps make this `radarbase/postgres:${POSTGRES_VERSION}-1`. This enables us to upload the image at a later stage, and increase the version suffix `-1` if we make changes to the image

File: dcompose-stack/radar-cp-hadoop-stack/optional-services.yml [hunk 3]
```diff
@@ -29,7 +29,7 @@ services:
   # RADAR Fitbit connector                                                     #
   #---------------------------------------------------------------------------#
   radar-fitbit-connector:
-    image: radarbase/kafka-connect-rest-fitbit-source:0.1.0
+    image: radarbase/kafka-connect-rest-fitbit-source:dev
```
Comment (blootsvoets) [inline dcompose-stack/radar-cp-hadoop-stack/optional-services.yml]: Use a tag here

Comment (nivemaham) [reply dcompose-stack/radar-cp-hadoop-stack/optional-services.yml]: Yes. I am waiting for the PR and release from fitbit-connector to be approved for this change.

**Pull request merged**